lib/propolis/src/intr_pins.rs

```diff
@@ -28,6 +28,13 @@ pub trait IntrPin: Send + Sync + 'static {
         }
     }
 
+    /// Set the state of this interrupt pin *without* treating the state change
+    /// as an edge in the interrupt line.
```

Comment:
I would have kept this as the top-line bit of the comment.  It's a nice summary to be backed up by the additional detail.

Reply:
Oh, okay, I think I misread your comment in https://github.com/oxidecomputer/propolis/pull/669#discussion_r1536254841 as suggesting that we replace that line with one about imports. I'll put it back!

Reply:
My fault for not making it clear in my comment.

Reply:
Put it back in https://github.com/oxidecomputer/propolis/pull/669/commits/2243baa115db2aa8152f1f047e9e0e56f9be35c4 --- what do you think?

Reply:
turbo nit: could probably do without the emphasis on the second `*without*`

otherwise... ship it

Reply:
good call, removed it. going to go ahead and merge now, since all the stuff
that's changed since the last CI build is docs.